SubmitReview/SubmitReview.test.ts

```diff
@@ -0,0 +1,101 @@
+import * as github from '@actions/github';
+import { LogTester } from '../tests/LogTester';
+import { createOctokitRestStub } from '../tests/OctokitRestStub';
+import { SubmitReview } from './SubmitReview';
+import { jiraClientStub } from '../tests/JiraClientStub';
+
+async function runAction(state: string, findEmailResult: string = null) {
+  github.context.payload = {
+    pull_request: {
+      number: 42,
+      title: "GHA-42 and GHA-43 are processed",
+    },
+    requested_reviewer: {
+      login: "test-user",
+      type: "User",
+    },
+    review: {
+      state
+    },
+    sender: {
+      login: 'test-user',
+      type: "User"
+    }
+  };
+  const action = new SubmitReview();
+  (action as any).jira = jiraClientStub;
+  (action as any).rest = createOctokitRestStub('GHA-42 and GHA-43 are processed');
+  (action as any).findEmail = async function (login: string): Promise<string> {
+    return findEmailResult;
+  };
+  await action.run();
+}
+
+describe('SubmitReview', () => {
+  const originalKeys = Object.keys(process.env);
+  let logTester: LogTester;
+
+  beforeEach(() => {
+    logTester = new LogTester();
+    for (const key of Object.keys(process.env)) {
+      if (!originalKeys.includes(key)) {
+        // Otherwise, changes form previous UT are propagated to the next one
+        delete process.env[key];
+      }
+    }
+    process.env['GITHUB_REPOSITORY'] = 'test-owner/test-repo';
+    process.env['INPUT_GITHUB-TOKEN'] = 'fake';
+  });
+
+  afterEach(() => {
+    logTester.afterEach();
+  });
+
+  it('Commented does nothing', async () => {
+    await runAction('commented');
+    expect(logTester.logsParams).toStrictEqual([
+      "Loading PR #42",
+      "Done",
+    ]);
+  });
+
+  it('Request Changes moves issue', async () => {
```

Comment:
These tests are valid if the dev has asked review from more than one person?
I mean one approval, request for changes should be enough to trigger the action right?

Reply:
It's triggered one by one, probably far apart in time. First comes is first served. The last review wins the state.

So if one reviewer approves, one requests changes, it depends on what happened last. And it will be moved there. And can be fixed manually if needed.